code/game/objects/structures/kitchen_spike.dm

```diff
@@ -98,8 +114,8 @@
 	victim.pixel_y = victim.get_standard_pixel_y_offset(180)
 	return TRUE
 
-
-/obj/structure/kitchenspike/user_buckle_mob(mob/living/M, mob/living/user) //Don't want them getting put on the rack other than by spiking
+/obj/structure/kitchenspike/user_buckle_mob(mob/living/M, mob/living/user)
+	// This will never run because meatspikes have density = TRUE
 	return
```

Comment:
```suggestion
	return 	// This will never run because meatspikes have density = TRUE
```
just put the comment on the return line,
also specify why you need to override this proc, if it's not being called why do you even need to override it in the first place